### PR TITLE
Feature/HS-29 Multi-lang via route path

### PR DESCRIPTION
**Pull request by urban-knight**
See more about translate module: https://ngstack-translate.netlify.com

### CODE REVIEW

File: src/app/app.routing.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { MainPageComponent } from './pages';
+
+
```
Comment (Jeddah96) [inline src/app/app.routing.ts]: Could you clean excess spaces?

File: src/app/pages/main-page/main-page.component.ts [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
-export class MainPageComponent {
-
-  public initSectionColor: string;
-
-  @ViewChild('navbar')
-  private navbar: NavbarComponent;
+export class MainPageComponent implements OnInit, OnDestroy {
+  constructor(
+    public fullpageService: MnFullpageService,
+    private route: ActivatedRoute,
```
Comment (Jeddah96) [inline src/app/pages/main-page/main-page.component.ts]: Private starts with `_`

File: src/app/pages/main-page/main-page.component.ts [hunk 3]
```diff
@@ -46,4 +48,15 @@ export class MainPageComponent {
...
+  ngOnInit() {
+    this.route.data.subscribe((d) => {
+      this.translate.activeLang = d.lang;
+      this.navbar.setActiveLang(d.lang);
+      this.titleService.setTitle('main.title');
+    });
+  }
+
+  ngOnDestroy(): void {
+    this.fullpageService.destroy('all');
```
Comment (Jeddah96) [inline src/app/pages/main-page/main-page.component.ts]: This is google-example implementation? I think, you should be investigate case with `unsubscribe` in `ngOnDestroy` life cycle hook.

File: src/assets/i18n/en.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+
```
Comment (Jeddah96) [inline src/assets/i18n/en.json]: Clean excess space

File: src/app/pages/main-page/main-page.component.ts (hunk 3 above)
Comment (urban-knight) [reply src/app/pages/main-page/main-page.component.ts]: This solution comes from original thread: https://github.com/meiblorn/ngx-fullpage/issues/451.
As the fullpage module would init each time the lang is changed, we have to destroy it before the page component reloalds with new lang via app router.
Thus it's the only cheap solution for ngx-fullpage while routing the app with different components and it comes w/o unsubscribe, as the fullpage-module isn't reloaded, but re-initialized.

**Pull request merged**